hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/CallFeature.java

```diff
@@ -106,7 +112,57 @@ public void createNewPrecompileTestContract() throws IOException {
     public void createNewEstimateTestContract() throws IOException {
         deployedContract = getContract(ESTIMATE_GAS);
         estimateContractAddress = deployedContract.contractId().toSolidityAddress();
+        admin = tokenClient.getSdkClient().getExpandedOperatorAccountId();
         receiverAccountId = accountClient.getAccount(AccountNameEnum.BOB);
+        receiverAccountAlias = receiverAccountId.getPublicKey().toEvmAddress().toString();
+    }
+
+    @Given("I mint a {string}")
+    public void mintNft(String tokenName){
+        var tokenId = tokenClient
+                .getToken(TokenClient.TokenNameEnum.valueOf(tokenName))
+                .tokenId();
+        networkTransactionResponse = tokenClient.mint(tokenId, RandomUtils.nextBytes(4));
+
+        verifyMirrorTransactionsResponse(mirrorClient, 200);
+
+        fungibleTokenId = tokenClient.getToken(FUNGIBLE).tokenId();
+        nonFungibleTokenId = tokenClient.getToken(NFT).tokenId();
+    }
+
+    @And("I approve and transfer 1 FUNGIBLE token to receiver account")
+    public void approveAndTransferFungibleTokenToReceiver(){
+        accountClient.approveToken(fungibleTokenId, receiverAccountId.getAccountId(), 1L);
+        networkTransactionResponse = tokenClient.transferFungibleToken(
+                fungibleTokenId,
+                tokenClient.getSdkClient().getExpandedOperatorAccountId(),
+                receiverAccountId.getAccountId(),
+                receiverAccountId.getPrivateKey(),
+                1L);
+        verifyMirrorTransactionsResponse(mirrorClient, 200);
+    }
+
+    @And("I approve and transfer 1 NFT token to receiver account")
+    public void approveAndTransferNftTokenToReceiver(){
+        var ntfId = new NftId(nonFungibleTokenId, 1L);
+        accountClient.approveNft(ntfId, receiverAccountId.getAccountId());
+        networkTransactionResponse = tokenClient.transferNonFungibleToken(
+                nonFungibleTokenId,
+                tokenClient.getSdkClient().getExpandedOperatorAccountId(),
+                receiverAccountId.getAccountId(),
+                List.of(1L),
+                receiverAccountId.getPrivateKey()
+        );
+        verifyMirrorTransactionsResponse(mirrorClient, 200);
+    }
+
+    @And("I associate {string} token to receiver account")
+    public void associateTokenToReceiver(String tokenName){
+        var tokenId = tokenClient
+                .getToken(TokenClient.TokenNameEnum.valueOf(tokenName))
+                .tokenId();
+        networkTransactionResponse = tokenClient.associate(receiverAccountId, tokenId);
```

Comment:
It would be shorter if we use the already existing `fungibleTokenId` and `nonFungibleTokenId`.
```suggestion
        tokenClient.associate(receiverAccountId, fungibleTokenId);
        networkTransactionResponse = tokenClient.associate(receiverAccountId, nonFungibleTokenId);
```